tests/pipeline/test_technical.py

```diff
@@ -174,3 +175,19 @@ def test_aroon_basic(self, lows, highs, expected_out):
         aroon.compute(today, assets, out, lows, highs)
 
         assert_equal(out, expected_out)
+
+
+class TestRateOfChangePercentage(ZiplineTestCase):
+    def test_rate_of_change_per(self):
+        rocp = RateOfChangePercentage(
+            inputs=(USEquityPricing.close,),
+            window_length=10
+        )
+        today = pd.Timestamp('2014')
+        assets = np.arange(5, dtype=np.int64)
+
+        data = np.ones((10, 5))
+        data[0, :] = np.full((1, 5), 2.0)
+        out = np.zeros(data.shape[1])
+        rocp.compute(today, assets, out, data)
+        assert_equal(out, np.full((5,), -50.0))
```

Comment:
Thanks for the good test case. Could you add one or two more cases as a regression test?